tasks/install_shell.sh

```diff
@@ -122,6 +122,12 @@ else
   fi
 fi
 
+if [ -n "$PT_retry" ]; then
+  retry=$PT_retry
+else
+  retry=5
+fi
+
```

Comment:
I would expect this to not be needed since there is a default value set in the `*.json` file (needs testing). If by any chance the `default` parameter is not being taken into consideration, then the powershell task should have this same logic in it and the `default` parameter from `*.json` files removed. It would only be confusing to have default value logic in tasks and `*.json` files.

Reply:
I removed the `default` value from `install_*json` files